src/main/java/io/leonis/subra/game/rule/BallInGoalRule.java

```diff
@@ -37,16 +37,8 @@
    * @return True if the ball is in the goal, false otherwise.
    */
   public boolean isBallInGoal(final Ball ball, final Goal goal) {
-    return Math.abs(ball.getX() - goal.getX())
-        < ((goal.getCardinalDirection().equals(CardinalDirection.NORTH)
-        || goal.getCardinalDirection().equals(CardinalDirection.SOUTH))
-        ? goal.getDepth() / 2f
-        : goal.getWidth() / 2f)
-        && Math.abs(ball.getY() - goal.getY())
-        < ((goal.getCardinalDirection().equals(CardinalDirection.EAST)
-        || goal.getCardinalDirection().equals(CardinalDirection.WEST))
-        ? goal.getDepth() / 2f
-        : goal.getWidth() / 2f);
+    return Math.abs(ball.getX() - goal.getX()) < goal.getDepth() / 2f
+        && Math.abs(ball.getY() - goal.getY()) < goal.getWidth() / 2f;
```

Comment:
@romnous Could you verify that this is actually the right way to verify this ?

Reply:
yes it is